aerospike-core/src/cluster/mod.rs

```diff
@@ -38,6 +33,10 @@ use self::partition_tokenizer::PartitionTokenizer;
 use crate::errors::{ErrorKind, Result};
 use crate::net::Host;
 use crate::policy::ClientPolicy;
+use aerospike_rt::RwLock;
```

Comment:
I think this would be better with normal std::sync::Mutex. As far as I can tell none of this stuff would ever be held over an await boundary, meaning leaving this stuff sync and relying on std::sync::MutexGuard being !Send is actually safer and more performant than fully asyncing it.

Reply:
Hi again.

Sorry, the property that we rely on to prevent holding locks over await boundaries is not because it's !Send.

Either way, I think the code would be more efficient with a regular mutex, which .